### PR TITLE
*: default to port 2379 for v3 commands

### PR DESCRIPTION
**Pull request by heyitsanthony**
set v3 endpoints to the etcd port so goreman + etcdctlv3 and benchmark won't
need --endpoints on every call

### CODE REVIEW

**Comment by xiang90** [comment]
lgtm

**Comment by heyitsanthony** [comment]
updated `etcd` too for completeness ptal /cc @xiang90

File: etcdmain/etcd.go [hunk 1]
```diff
@@ -229,7 +229,11 @@ func startEtcd(cfg *config) (<-chan struct{}, error) {
 		plog.Infof("clientTLS: %s", cfg.clientTLSInfo)
 	}
 	clns := make([]net.Listener, 0)
-	for _, u := range cfg.lcurls {
+	lcurls := cfg.lcurls
+	if cfg.v3demo {
+		lcurls = nil
```
Comment (xiang90) [inline etcdmain/etcd.go]: when we enable v3-demo we still want v2 works i think.

**Comment by xiang90** [comment]
probably for now, just make 2378 work for gRPC (also in the v3demo goreman file)? we will make them on the same port work soon.

**Comment by heyitsanthony** [comment]
ok

**Comment by xiang90** [comment]
looks good.

**Pull request merged**